build-system/erbb/generators/faust/code_template.hpp

```diff
@@ -79,6 +99,7 @@ void  %module.name%::init ()
    mydsp::fManager = &mem;
 
    mydsp::classInit (erb_SAMPLE_RATE);
+   call_faust_mydsp_memoryCreate_if_exists (dsp, 0);
```

Comment:
This looks good and works with the 2.50.6 build of Faust I have installed.
The VCV rack plugin now builds, loads and functions as expected.  I haven't had a chance to test it on the hardware but the daisy binary builds fine.